extensions/wikia/FacebookClient/FacebookClientFactory.php

```diff
@@ -27,7 +27,12 @@ public function connectToFacebook( $wikiaUserId, $fbUserId ) {
 				$status->setResult( false );
 				$status->error( 'fbconnect-error-already-connected' );
 			} else {
-				$map = \FacebookMapModel::createUserMapping( $wikiaUserId, $fbUserId );
+				$bizToken = FacebookClient::getInstance()->getBizToken();
```

Comment:
There's no namespace defined in this class, and none of the classes currently with ```\``` define special namespaces so its superfluous.  Its not part of our style guide but the rest of the file uses it, so I guess its better to keep consistent.